dt, recipes-kernel: Add hdmi support

drozd-svtln opened the pull request:
Changes: 
- Add the device tree overlays for hdmi, modify the recipe to add the hdmi
dtbos.
- Add the memory range at 0x2000000 to xen static-mem to fix rpi firmware issues

Do not merge before: https://github.com/xen-troops/meta-xt-rpi5/pull/42

oleksiimoisieiev commented:
```
---------------------------------------------
0001-dt-recipes-kernel-Add-hdmi-support.patch
---------------------------------------------
WARNING: added, moved or deleted file(s), does MAINTAINERS need updating?
#43: 
new file mode 100644

WARNING: From:/Signed-off-by: email name mismatch: 'From: svitlana_drozd <svitlana_drozd@epam.com>' != 'Signed-off-by: Svitlana Drozd <svitlana_drozd@epam.com>'

total: 0 errors, 2 warnings, 474 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

0001-dt-recipes-kernel-Add-hdmi-support.patch has style problems, please review.

```

Commit:
recipes-kernel: Add hdmi support

Add the device tree overlays for hdmi, modify the recipe to add the hdmi
dtbos.
These changes cause the rpi firmware to fail, but the issue will be
resolved by the following commit.

Signed-off-by: Svitlana Drozd <svitlana_drozd@epam.com>
Reviewed-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>

Commit:
dt: add memory range at 0x2000000 to xen static-mem to fix firmware

Firmware is crashing with
```
[    0.998420] raspberrypi-firmware passthrough:firmware: Request
0x00030066 returned status 0x00000000
[    1.008771] vc4-drm passthrough:axi:gpu: [drm] Couldn't stop firmware
display driver: -22
```
The RPI_FIRMWARE_STATUS is not changing after an interrupt and a
callback from the mailbox.

This happens because the firmware can only work with memory
addresses below 1Gb memory, so to fix that, a chunk of memory in this
range was added to xen.

Signed-off-by: Svitlana Drozd <svitlana_drozd@epam.com>
Reviewed-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>

drozd-svtln commented:
Fixed email name in both commits

oleksiimoisieiev commented:
Please rebase on top of the latest changes, Fix email warining and rename commit "  dt, recipes-kernel: Add hdmi support" to " recipes-kernel: Add hdmi support"

LGTM `Reviewed-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>`

drozd-svtln commented:
Fixed warning, renamed commit, added trailer
